deeppavlov/models/morpho_tagger/common.py

```diff
@@ -56,7 +56,8 @@ def predict_with_model(config_path: [Path, str], infile: Optional[Union[Path, st
         else:
             data = sys.stdin.readlines()
     model = build_model(config, load_trained=True)
-    model.pipe[-1][-1].set_format_mode(output_format)
+    if isinstance(model.pipe[-1][-1], TagOutputPrettifier):
+        model.pipe[-1][-1].set_format_mode(output_format)
```

Comment:
It forces to always use `tag_output_prettifier` last in pipe.
Also, why not to set `output_format` as argument to `__init__` of `TagOutputPrettifier`

Reply:
Now this line of code iterates over the components.